cloudbuild/periodic/run-all-tests.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+set -uxo pipefail
+
+COMMIT=$(git rev-parse HEAD)
+TESTS_TO_RUN=(
+  ":DataprocInitActionsTestSuite"
+) 
+
+run_tests() {
+  bazel test \
```

Comment:
I think that we should reuse the same script and Cloud Build config for running presubmit and nightly tests. We just need to add a parameter that will force it to run all tests and upload results to GCS. After that we can have a separate workflow that will read test results from GCS and transform them in testgrid format.

Reply:
I like having the GCB workflows separate because this way we can easily distinguish between presubmit and periodic runs when going through our GCB workflow history. This makes it easier for tracking and debugging. We also probably only want the periodic test results to show in TestGrid - we're not interested in having the presubmit results in TestGrid, correct? In this case, the transformation should only happen for the periodic tests' results.

What do you think?

Reply:
After discussion, I consolidated both workflows/triggers to be controlled by a single yaml config. The _RUN_ALL_TESTS controls if it's run as a presubmit or a periodic test.

Also, the build number is now extracted by reading GCS and finding the folder name (i.e. build number) with the highest number.